src/views/ProductCatalogWrapper.vue

```diff
@@ -168,6 +166,7 @@ export default defineComponent({
 
     onBeforeMount(async () => {
       await loadAppearance()
+      await fetchProducts()
```

Comment:
Can we load these two in parallel, that is without `await` (or using `await Promise.all([...])`)?

Reply:
Makes sense!